common.js

```diff
@@ -1939,6 +1939,32 @@ const allSoloCars = {
       '1993': ['dsp'],
     },
   },
+  'Morgan': {
+    'Plus 8': {
+      'all': ['as', 'fp'],
+    },
+    'Roadster (V6)': {
+      'all': ['as'],
+    },
+    '4/4 & Plus 4': {
```

Comment:
Is this the same as the +4 later on? If so you should delete this and add `es` to the other 4/4 and +4 models.